packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -282,6 +282,17 @@ export const CODEPIPELINE_CROSS_ACCOUNT_KEY_ALIAS_STACK_SAFE_RESOURCE_NAME = '@a
  */
 export const S3_CREATE_DEFAULT_LOGGING_POLICY = '@aws-cdk/aws-s3:createDefaultLoggingPolicy';
 
+/**
+* Enable this feature flag to restrict the decryption of a SQS queue, which is subscribed to a SNS topic, to
```

Comment:
Can you also add this info [here](https://github.com/aws/aws-cdk/blob/68f09b7c6f8e6c1ddf13fdd4e116d12333d24c46/packages/%40aws-cdk/cx-api/README.md?plain=1#L14)? We haven't found the best way to document these yet so until then I figure that will have to do.

Reply:
If it is ok I will add a chore later and update the feature-flag documentation :)